src-v2/NTS.Domain.Setup/Entities/Competition.cs

```diff
@@ -24,36 +25,40 @@ public static Competition Update(
     private Competition(int id) : base(id) { }
     private Competition(
         int id, 
-        string name, 
-        CompetitionRuleset type,
+        string name,
+        CompetitionRuleset ruleset,
+        CompetitionType type,
         DateTimeOffset startTime,
         int? criRecovery,
         IEnumerable<Phase> phases,
-        IEnumerable<Contestant> contestants) : this(name, type, startTime, criRecovery)
+        IEnumerable<Contestant> contestants) : this(name, ruleset, type, startTime, criRecovery)
     {
         Id = id;
         _phases = phases.ToList();
         _contestants = contestants.ToList();
     }
-    private Competition(string name, CompetitionRuleset type, DateTimeOffset startTime, int? criRecovery)
+    private Competition(string name, CompetitionRuleset ruleset, CompetitionType type, DateTimeOffset startTime, int? criRecovery)
     {
         if (type == default)
         {
-            throw new DomainException(nameof(type), "Competition Type is required");
+            //CompetitionType defualt is enabled to ensure EmsImportBehind function correctly
+            //throw new DomainException(nameof(type), "Competition Type is required");
```

Comment:
Is it not possible to avoid disabling validation? Why not just pass the argument?

Reply:
Currently, the EmsImportBehind calls `Competition.Create` with a default value for `type` this will always cause the EmsImport to throw an exception.